Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -112,10 +115,25 @@ class KernFeatureWriter(BaseFeatureWriter):
     def setContext(self, font, feaFile, compiler=None):
         ctx = super().setContext(font, feaFile, compiler=compiler)
         ctx.gdefClasses = self.getGDEFGlyphClasses()
-        ctx.kerning = self.getKerningData(font, feaFile, self.getOrderedGlyphSet())
+        ctx.glyphSet = self.getOrderedGlyphSet()
 
-        feaScripts = ast.getScriptLanguageSystems(feaFile)
-        ctx.scriptGroups = self._groupScriptsByTagAndDirection(feaScripts)
+        # TODO: Also include substitution information from Designspace rules to
+        # correctly the scripts of variable substitution glyphs, maybe add
```

Comment:
```suggestion
        # correctly set the scripts of variable substitution glyphs, maybe add
```

Reply:
maybe file an issue about the fact that designspace rule substitutions don't get taken into account when doing the GSUB closure to determine the split-by-script kerning